CMS-Enterprise/easi-app
# EASI-4552 Manage system team

# EASI-4552
https://www.figma.com/design/z8GeZPDfIh9oKfHK1cX7Fo/System-centric-EASi?node-id=1308-66825&node-type=section&t=FWxU1EFHza18WQ1A-0
## Description

Modifies the team management components from System Profile to fit the System Workspace context.


## How to test this change

- Check that the team management functionality works in the new workspace context, while the original System Profile version still works in the same way.
- Preview locally with mock data to check role requirement conditions
```
// src/views/SystemProfile/index.tsx
import teamRoles from 'data/mock/workspaceTeamRoles';
import teamRequisiteRoles from 'data/mock/workspaceTeamRoles';
...
- const usernamesWithRoles = getUsernamesWithRoles(personRoles);
+ const usernamesWithRoles = getUsernamesWithRoles(teamRoles);
or
+ const usernamesWithRoles = getUsernamesWithRoles(teamRequisiteRoles);
```
- Unit tests were made to test role requirements for the workspace mode

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
